content/articles/data-access-with-ado-dotnet-in-micro-focus-visual-cobol/index.md

```diff
@@ -0,0 +1,217 @@
+
+It's essentially a set of access and manipulation processes that a developer can utilize to gain access to his or her data.
+
+The best part is that you don't need to install anything, because it's already included in the `.NET` Framework.
+
+### Prerequisite
+- Install [Visual Studio Code](https://code.visualstudio.com/).
+- Add [Microfocus COBOL extension](https://marketplace.visualstudio.com/items?itemName=Micro-Focus-AMC.mfcobol) to Visual Studio code editor.
+- Have some background information on database management.
+
+### Table of contents
+- [Objective](#objective)
+- [What is data object?](#what-is-a-data-object)
+- [Basic ADO flow](#basic-ado-flow)
+- [Preparation](#preparation)
+- [Create a Connection](#create-a-connection)
+- [Create a Command](#create-a-command)
+- [Read the Data](#read-the-data)
+- [Housekeeping](#housekeeping)
+- [Other Data Providers](#other-data-providers)
+- [Wrap-Up](#wrap-up)
+
+### Objective
+By the end of this tutorial, the reader will have understood:
+- How to establish a link to the data objects in ADO.NET.
+- Create commands that will help to access the data objects using Micro Focus Visual Studio.
+
+### What is a data object
+A data object is indeed a storage area that holds a value or a set of values. Each value could be accessed by using the object's identifier like a variable.
+
+Furthermore, each object has its data type.
+
+For example:
+- Products, sales, and customer are examples of data objects of a sales database
+- Patients may be objects of a medical database
+- Students, professors, and courses may be objects of an University database.
+
+### Basic ADO flow
+We will start by looking what `SQL Server's namespace` are, which governs data access.
+
+At the end of this article, we will go over different data providers.
+
+A namespace is a set of symbols often used to distinguish and relate objects of different types.
+
+As a result of a namespace, every particular collection of objects will have unique names, allowing them to be identified easily.
+
+> `System.Data.SqlClient` is the namespace to remember if you want to connect to SQL Server. The `.NET` data provider for SQL server is located in this namespace. 
+
+To connect to SQL servers in `.NET` framework, you will start by writing the namespace:
+
+```
+using System.Data;
+using System.Data.SqlClient;
+```
+
+> Keep this namespace in mind. It's critical!
+
+There are presently 16 classes in the SQLClient namespace. These classes provide a programmatic way to establish connections to databases, verify permissions, manage exceptions, and do other duties related to data access.
+
+Let's get started using ADO.
+
+To view data object, follow this basic procedure:
+1. Establish a link to the data object.
+2. Make a command that the data object will follow.
+3. Consider the data that the data object has provided.
```

Comment:
Could you please rephrase these 2 sentences? These sentences doesn't convey any meaning to me.